test/nn/test_convolution.py

```diff
@@ -2160,18 +2159,19 @@ def helper(n, c, h, w, out_channels, kernel_size, dilation, groups, input_format
 
     @onlyCUDA
     @skipCUDAIfRocmVersionLessThan((4, 3))
-    @skipCUDAIfNotMiopenSuggestNHWC
     @skipCUDAIfCudnnVersionLessThan(7603)
+    # randint and randint_like with dtype=torch.cfloat raises
+    # RuntimeError: check_random_bounds handles only integral, floating-point and boolean types
     @dtypes(torch.half, torch.float, torch.cfloat)
     def test_conv_cudnn_nhwc(self, device, dtype):
         def helper(n, c, h, w, out_channels, kernel_size, groups):
-            input = torch.randint(-3, 3, (n, c, h, w), dtype=dtype, device=device)\
-                .to(memory_format=torch.channels_last)
+            input = torch.randint(-3, 3, (n, c, h, w), device=device)\
+                .to(memory_format=torch.channels_last, dtype=dtype)
```

Comment:
This looks semantically equivalent to the previous statement, but adding a line or two explaining why this is necessary would be nice

Reply:
lines 2163 and 2164 are the comment explaining the change